TimePlatformer/Assets/Scripts/EnemyHorizontalMovementScript.cs

```diff
@@ -0,0 +1,45 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class EnemyHorizontalMovementScript : MonoBehaviour {
+
+    [Range(-1, 1)] public int direction;
+
+    public float walkSpeed;
+
+    public float maxRange;
+
+    private Vector2 startPosition;
+
+    private Vector2 walkingVector;
+
+	// Use this for initialization
+	void Start () {
```

Comment:
Please put opening braces on the next line. (Allman style, as opposed to K&R style)